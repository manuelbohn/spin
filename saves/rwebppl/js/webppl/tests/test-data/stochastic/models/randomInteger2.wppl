var model = function() {
  var x = randomInteger(2);
  factor(x === 0 ? 0 : Math.log(0.25));
  return x;
};
