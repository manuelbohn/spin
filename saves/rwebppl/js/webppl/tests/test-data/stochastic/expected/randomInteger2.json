{
  "hist": {
    "0": 0.8,
    "1": 0.2
  }
}
